backend/game/engine/player_queue.py

```diff
@@ -1,25 +1,21 @@
 import itertools
 
+from game.player.player import Player
+
 
 class GamePlayers():
 
     def __init__(self):
         self.all_players = []
-        self.dead_players = []
-        self.alive_players = []
         self.current_player = None
 
-    def _add_initial_player(self, player):
-        self.all_players.append(player)
-
-    def _remove_initial_player(self, player):
-        if player in self.all_players:
-            self.all_players.remove(player)
+    def _add_player_to_game(self, new_player):
+        if isinstance(new_player, Player):
```

Comment:
Again, not pythonic. If it's not an instance of that thing, then raise an error. Otherwise, move on unindented.